spyder_kernels/console/kernel.py

```diff
@@ -346,14 +350,21 @@ def get_var_properties(self):
             return None
 
     @comm_handler
-    def get_value(self, name):
+    def get_value(self, name, encoded=False):
```

Comment:
Why `encoded` is `False` by default and not `True`? Don't we need all values to be encoded with Cloudpickle?

Reply:
get_value is also used in spyder_kernels e.g. in the tests where there is no need for encoding. Only the comm call requires encoding. The extra bonus is that it makes the encoding explicit on the frontend side